### PR TITLE
Adding new state for ignored submitted orders

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
This is mainly follow up on comments in #115 where we decided to have a separate state for submitted orders that get expired.

I also talked to @joanna20 about not automatically refund already approved orders (orders that were approved by partners but haven't been fulfilled yet). We decided that we are not going to automatically refund them but we may want to notify internal admins to contact partners and follow up (or eventually send follow up emails).

# Changes
I added new `SELLER_LAPSED` state (which im still struggling with the name a little) based on https://www.notion.so/artsy/37c311363ef046c3aa546047e60cc58a?v=de68d5bbc30748f88b0d92a059bc0ba8. We talked a bit about having `expired` state for all possible reasons an order goes to expired, but Im not opposed to have more specific state in this case mainly knowing we are going to post events and for the consumer of these events they may want to defer between an order expired because of seller never responded or order expired because never fulfilled or other possible expired cases.

I also renamed `ExpireOrderJob` to `OrderFollowUpJob` since in Approve case we are actually not expiring anything and we just want to post an event. Let me know what you think.

### CODE REVIEW

**Comment by joanna20** [comment]
@ashkan18 does this state apply to either step - the order confirm or the order fulfill step? Can an order get into this state if the seller doesn't confirm within 2 days, and also if they don't ship within 7?

**Comment by ashkan18** [comment]
@joanna20 no actually, sorry for my bad description, it only applies to submitted orders that are not approved/rejected by partners within 2 days. 
For the case of approved orders that are not fulfilled state will stay approved but im going to post an event so we can nudge admins or possibly partners about it.

**Comment by ashkan18** [comment]
cc: @madeleineb

File: app/services/order_service.rb [hunk 1]
```diff
@@ -69,29 +69,42 @@ def self.fulfill_at_once!(order, fulfillment, by)
...
+
   def self.abandon!(order)
     order.abandon!
   end
 
   def self.valid_currency_code?(currency_code)
     Order::SUPPORTED_CURRENCIES.include?(currency_code.downcase)
   end
+
+  def self.refund(order)
```
Comment (williardx) [inline app/services/order_service.rb]: We should be rescuing `PaymentError` here and logging an unsuccessful refund if it happens

Comment (williardx) [reply app/services/order_service.rb]: ah! I see, you're still catching it in `reject!`

File: spec/jobs/order_follow_up_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+require 'timecop'
+
+describe OrderFollowUpJob, type: :job do
+  let(:state) { Order::PENDING }
+  let(:order) { Fabricate(:order, state: state) }
+  describe '#perform' do
+    context 'with an order in the same state after its expiration tim' do
```
Comment (williardx) [inline spec/jobs/order_follow_up_job_spec.rb]: Typo - should be "time"

**Comment by ashkan18** [comment]
As part of this also updated shipping expiration to 7 days, cc: @madeleineb @joanna20 

@joeyAghion going to merge this, I'll address comments in separate PR.

**Pull request merged**